mlflow/crewai/__init__.py

```diff
@@ -0,0 +1,169 @@
+"""
+The ``mlflow.crewai`` module provides an API for tracing CrewAI AI agents.
+"""
+
+from mlflow.crewai.autolog import (
+    patched_class_call,
+)
+from mlflow.utils.annotations import experimental
+from mlflow.utils.autologging_utils import autologging_integration, safe_patch
+
+FLAVOR_NAME = "crewai"
+
+
+@experimental
+@autologging_integration(FLAVOR_NAME)
+def autolog(
+    log_traces: bool = True,
+    disable: bool = False,
+    silent: bool = False,
+):
+    """
+    Enables (or disables) and configures autologging from CrewAI to MLflow.
+    Only synchronous calls are supported. Asynchnorous APIs and streaming are not recorded.
```

Comment:
Q: Is streaming available in CrewAI?

Reply:
To my understanding, it's not supported.

Reply:
yea, can we remove it from thd docstring?